content/en/docs/tasks/security/authorization/authz-custom/index.md

```diff
@@ -0,0 +1,236 @@
+---
+title: External authorization with custom action
+description: Shows how to integrate and delegate access control to external authorization system.
+weight: 35
+keywords: [security,access-control,rbac,authorization,custom, opa, oauth, oauth2-proxy]
+owner: istio/wg-security-maintainers
+test: no
+---
+
+This task shows you how to set up Istio authorization policy with the `CUSTOM` action to delegate the access control to
+an external authorization system. This can be used to integrate with [OPA authorization](https://www.openpolicyagent.org/docs/latest/envoy-authorization/),
+[oauth2-proxy](https://github.com/oauth2-proxy/oauth2-proxy), your own custom external authorization server and more.
+
+{{< boilerplate experimental-feature-warning >}}
+
+## Before you begin
+
+Before tackling this task you must perform the following actions:
+
+* Read the [authorization concept](/docs/concepts/security/#authorization).
+
+* Follow the [Istio installation guide](/docs/setup/install/istioctl/) to install Istio.
+
+* Deploy test workloads:
+
+    This task uses two workloads, httpbin and sleep, deployed on one namespace, foo.
+    Both workloads run with an Envoy proxy in front of each. Deploy the example namespace
+    and workloads with the following command:
+
+    {{< text bash >}}
+    $ kubectl create ns foo
```

Comment:
Don't you need to add label to enable sidecar injection?

Reply:
Updated.